openid-4-verifiable-presentations-1_0.md

```diff
@@ -721,25 +721,17 @@ Within the particular `claims` array, the same `id` MUST NOT
 be present more than once.
 
 `path`:
-: REQUIRED if the Credential Format uses a JSON-based claims structure; MUST NOT
+: REQUIRED if the Credential Format uses a JSON-based claims structure (e.g., IETF SD-JWT VC, W3C Verifiable Credentials and Anon Creds); MUST NOT
```

Comment:
Isn't the AnonCreds data structure essentially a name/value list?

Reply:
```suggestion
: REQUIRED if the Credential Format uses a JSON-based claims structure (e.g., IETF SD-JWT VC and W3C Verifiable Credentials); MUST NOT
```